components/ArticleMenu/ArticleMenu.tsx

```diff
@@ -139,7 +139,7 @@ const ArticleMenu = ({
               }}
             >
               <HeartIcon
-                className={`w-6 h-6${
```

Comment:
Are these swapped automatically? I just looked at another PR and it was the same?